packages/generator-langium/langium-template/src/extension.ts

```diff
@@ -34,12 +33,15 @@ function startLanguageClient(context: vscode.ExtensionContext): LanguageClient {
         debug: { module: serverModule, transport: TransportKind.ipc, options: debugOptions }
     };
 
+    const fileSystemWatcher = vscode.workspace.createFileSystemWatcher('**/*.<%= file-glob-extension %>');
+	context.subscriptions.push(fileSystemWatcher);
```

Comment:
Nitpick: inconsistent whitespace

Reply:
Thanks, somehow a tab character found it's way into there 🤔